### PR TITLE
Update app.component.html

### PR DESCRIPTION
**Pull request by CindyHXH**

### CODE REVIEW

**Comment by CindyHXH** [comment]
@Shanfang  this pull request is for travis ci. The build information can be found at [build](https://travis-ci.org/CindyHXH/SmartNote/builds)

**Review by Shanfang — COMMENTED** [review]
Works perfectly👍

File: smart-note/e2e/about/about.po.ts [hunk 1]
```diff
@@ -18,12 +18,12 @@ export class About {
   
   // get info
   getAppInfo() {
-    return element(by.css('.about.profile-card.header > h2')).getText();
+    return element(by.id('app-info')).getText();
```
Comment (Shanfang) [inline smart-note/e2e/about/about.po.ts:21]: It much easier to use id to select an element. Life saving!

File: smart-note/e2e/addnote/addnote.e2e-spec.ts [hunk 2]
```diff
@@ -1,22 +1,55 @@
...
+        var options = element.findElements(by.css('my-dropdown-menu'))   
+        .then(function(options){
+            options[optionNum].click();
+        });
+    }
+};
 
-// describe('Addnote page', () => {
-//   let page: Addnote;
+describe('Addnote page', () => {
```
Comment (Shanfang) [inline smart-note/e2e/addnote/addnote.e2e-spec.ts:18]: Good job, this fully test the addnote component, but some of the edge cases are not test yet, like saving a note without adding a tag, saving a note without adding any content, etc.

File: smart-note/e2e/addnote/addnote.po.ts [hunk 3]
```diff
@@ -9,4 +9,19 @@ export class Addnote {
   getTitle() {
     return browser.getTitle();
   }
+  getNoteTitle() {
```
Comment (Shanfang) [inline smart-note/e2e/addnote/addnote.po.ts:12]: Good! This test the tag thoroughly.

File: smart-note/e2e/login/login.e2e-spec.ts [hunk 4]
```diff
@@ -3,45 +3,83 @@ import { browser } from 'protractor';
...
+  it('should get the title', () => {   
     expect(page.getTitle()).toEqual('SmartNote');
   });
 
-  // get user name
-  it('should get username', () => {
-    //page.navigateTo();
-    var username = page.getUsername().sendKeys('demo3');
+  it('should check invalid username or password', () => {
+    let username = page.getUsername();
```
Comment (Shanfang) [inline smart-note/e2e/login/login.e2e-spec.ts:27]: Nice job, this will ensure that unauthorized user is not allowed to access the notes.

**Review by Shanfang — COMMENTED** [review]
This e2e test does the work!
